driver-core/src/main/com/mongodb/internal/ClientSideOperationTimeout.java

```diff
@@ -75,7 +75,7 @@ public boolean hasTimeoutMS() {
      * @return true if the timeout has been set and it has expired
      */
     public boolean expired() {
```

Comment:
We should change this to hasExpired at some point - just to keep the nomenclature consistent. Doesn't have to be done in this PR.